node/worker_threads.ts

```diff
@@ -54,7 +55,7 @@ class _Worker extends EventEmitter {
     const handle = this[kHandle] = new Worker(
       specifier,
       {
-        ...(options || {}),
+        name: PRIVATE_WORKER_THREAD_NAME,
```

Comment:
This will cause all workers to have the same name; maybe we could add some kind of counter?

Reply:
Cross worker counter seems like a pain, I thought about `Date.now()` but that's unreliable as well...

The name doesn't serve another purpose and is deleted by the worker on startup, it's probably fine.